### PR TITLE
[5.0] Fix Multi-Row Insert For SQLite

### PR DESCRIPTION
**Pull request by konomae**
Multi-row insert does not work with same data.
Use `UNION ALL` instead `UNION`.

``` php
// DB connection
'sqlite' => [
    'driver'   => 'sqlite',
    'database' => ':memory:',
    'prefix'   => '',
];

class Post extends \Eloquent {}

// 3 insertion
Post::insert([
    ['test' => 1],
    ['test' => 1],
    ['test' => 1]
]);

// Result
Post::all()->count(); // => 1
```

``` bash
$ sqlite3
```

``` sql
create table test (column integer);

-- different data, 3 records inserted
insert into "test" ("column") select 1 as "column" union all select 2 as "column" union all select 3 as "column";


-- same data with `UNION ALL`, 3 records inserted
insert into "test" ("column") select 4 as "column" union all select 4 as "column" union all select 4 as "column";

-- same data with `UNION`, only one record inserted
insert into "test" ("column") select 5 as "column" union select 5 as "column" union select 5 as "column";

-- result
select * from test;
1
2
3
4
4
4
5
```

### CODE REVIEW

**Comment by GrahamCampbell** [comment]
Please include tests.

**Comment by konomae** [comment]
@GrahamCampbell I pushed a commit including tests.

File: tests/Database/DatabaseEloquentIntegrationTest.php [hunk 1]
```diff
@@ -242,6 +242,30 @@ public function testBasicMorphManyRelationship()
 		$this->assertEquals('First Post', $photos[3]->imageable->name);
 	}
 
+	public function testMultiInsertsWithDifferentValues()
```
Comment (GrahamCampbell) [inline tests/Database/DatabaseEloquentIntegrationTest.php]: Please make sure the test methods are separated by two blank lines, not one.

**Comment by konomae** [comment]
Blank line style fixed

**Pull request merged**